.github/workflows/pr-lint.yml

```diff
@@ -50,5 +50,15 @@ jobs:
           LABEL_NAMES: ${{ toJson(github.event.pull_request.labels.*.name) }}
         if: contains(env.LABEL_NAMES, 'under review') == contains(env.LABEL_NAMES, 'in progress')
         run: |
-          echo "PR must have "exactly one" of these labels: ['under review', 'in progress']."
-          exit 1
+          # Filter labels starting with "status:"
+          STATUS_LABELS=$(echo $LABEL_NAMES | jq -r '.[] | select(startswith("status:"))')
+          STATUS_COUNT=$(echo "$STATUS_LABELS" | wc -l)
+          
+          echo "Found status labels: $STATUS_LABELS"
+          echo "Count of status labels: $STATUS_COUNT"
+          
+          if [ "$STATUS_COUNT" -ne 1 ]
+          then
+            echo "PR must have exactly one label starting with 'status:'. Found $STATUS_COUNT."
+            exit 1
+          fi
```

Comment:
We don't need this much change here I think

Reply:
We just need to replace the old labels, I don't know why we changed the entire logic

Reply:
> We just need to replace the old labels, I don't know why we changed the entire logic

I decided that almost all status labels are valid and can pass the PR lint. Are you suggesting allowing only the 'pending review' and 'in progress' statuses to pass the lint check?

Reply:
Label validation logic was simple: requiring one of "in progress" or "under review" labels but not both at the same time. We can update this logic with new ones + blocked label.

Reply:
> We can update this logic with new ones + blocked label.

That what I wanted to do, allow new status labels except invalid statuses, just without rewriting all statuses manually.
But I think now, its better to be explicit with labels in pr lint for the case if we add new status labels in the future

Reply:
I didn't mean all new labels, what I mean is these 3 labels:

- status: pending review
- status: blocked
- status: in progress